nb-extension/js/widgets/DeclWidgetModel.js

```diff
@@ -1,46 +1,41 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-define(function() {
+define(["nbextensions/widgets/widgets/js/widget"], function(Widget) {
```

Comment:
This path is what `bqplot` uses. The ipywidgets seem to have added an alias to use the string 'jupyter-js-widgets' https://github.com/ipython/ipywidgets/pull/414/files#diff-13adbd6bc145945c1c3bd7508e2c46ddR69